source/renderer/app/stores/ProfileStore.js

```diff
@@ -302,10 +304,16 @@ export default class ProfileStore extends Store {
   _acceptTermsOfUse = async () => {
     await this.setTermsOfUseAcceptanceRequest.execute();
     await this.getTermsOfUseAcceptanceRequest.execute();
+    await enableTermsAgreedUserMenuItemsChannel.send();
   };
 
-  _getTermsOfUseAcceptance = () => {
-    this.getTermsOfUseAcceptanceRequest.execute();
+  _getTermsOfUseAcceptance = async () => {
+    await this.getTermsOfUseAcceptanceRequest.execute();
+    if (this.getTermsOfUseAcceptanceRequest.result) {
+      await enableTermsAgreedUserMenuItemsChannel.send();
+    } else {
+      await disableTermsAgreedUserMenuItemsChannel.send();
```

Comment:
@yakovkaravelov please rename this channel to: `disableApplicationMenuNavigationChannel`